plugins/jobtype/src/azkaban/jobtype/HadoopSecurePigWrapper.java

```diff
@@ -149,6 +155,47 @@ public static void runPigJob(String[] args) throws Exception {
     throw new RuntimeException("Pig job failed.");
   }
 
+  /**
+   * Dump Hadoop counters for each of the M/R jobs in the given PigStats.
+   * 
+   * @param pigStats
+   */
+  private static void dumpHadoopCounters(PigStats pigStats) {
+    try {
+      if (props.getBoolean("pig.dump.hadoopCounter", false)) {
+        if (pigStats != null) {
+          JobGraph jGraph = pigStats.getJobGraph();
+          Iterator<JobStats> iter = jGraph.iterator();
+          while (iter.hasNext()) {
```

Comment:
May be a bit far fetched. Suppose if we have some mapper failures and Hadoop re-executes them, then are we going to have pig counters from that in the logs ?